pjlink.js

```diff
@@ -225,14 +225,34 @@ instance.prototype.actions = function(system) {
 		'shutterOpen':    { label: 'Open Shutter' },
 		'shutterClose':   { label: 'Close Shutter' },
 		'freeze':         { label: 'Freeze Input' },
-		'unfreeze':       { label: 'Unfreeze Input' }
-
+		'unfreeze':       { label: 'Unfreeze Input' },
+		'inputToggle': {
+			label: 'Toggle Input',
```

Comment:
Isn't "change input", "set input" or "select input" better name? Toggling usually infers that it is a boolean choice.

Reply:
The [specification](https://pjlink.jbmia.or.jp/english/data_cl2/PJLink_5-1.pdf) calls the command "Input switch" - this should be used in my opinion. I also would use "toggle" for a boolean choice only.